lib/oidc/types/api.ts

```diff
@@ -120,6 +120,18 @@ export interface OriginalUriApi {
   removeOriginalUri(state?: string): void;
 }
 
+export interface DPoPRequest {
+  url: string;
+  method: string;
+  nonce?: string;
+  accessToken?: AccessToken;
+}
+
+export interface DPoPHeaders {
+  Authorization: string;
+  Dpop: string;
```

Comment:
Does the case need to change?

```suggestion
  DPoP: string;
```